org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/ProjectsManager.java

```diff
@@ -363,7 +364,7 @@ public static IProject createJavaProject(IProject project, IPath projectLocation
 
 		IJavaProject javaProject = JavaCore.create(project);
 		configureJVMSettings(javaProject);
-
+		JobHelpers.waitForJobsToComplete();
```

Comment:
Do we know what kind of jobs are we waiting here?

Reply:
Eclipse AutoBuildJob

Reply:
Would it be safer to just use `JobHelpers.waitForBuildJobs(MAX_TIME_MILLIS);` as opposed to this method which does additional things ?